src/scss/includes/reviewScore.scss

```diff
@@ -1,10 +1,11 @@
 .reviewScore {
-  display: inline-block;
-  color: #5c6f84;
-  margin: 2px 10px 2px 0;
+  color: $secondary_text;
+  display: inline-flex;
+  justify-content: center;
```

Comment:
This rule probably doesn't have any effect as it's an inline element without a specified width.

Reply:
Yes, only `align-items` is necessary.